### PR TITLE
UHM-3609: Add test status to labOrderList Page 

### PR DESCRIPTION
**Pull request by jomadoye**
screen shot 2018-11-09 at 9 37 02 am

### CODE REVIEW

File: app/js/components/LabsStatusDisplay.jsx [hunk 1]
```diff
@@ -0,0 +1,107 @@
+import React, { PureComponent } from 'react';
+import R from 'ramda';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import axios from '../config/axios';
+// import { formValidations } from '@openmrs/react-components';
```
Comment (rhenshaw56) [inline app/js/components/LabsStatusDisplay.jsx]: can you remove the commented bit

File: app/js/components/LabsStatusDisplay.jsx [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+    };
+
+    this.getTestStatus = this.getTestStatus.bind(this);
+  }
+
+  async componentWillMount() {
+    const {
+      conceptUUID, patientUUID,
+    } = this.props;
+    const encounters = await axios.get(`encounter?s=byObs&patient=${patientUUID}&obsConcept=${conceptUUID}&v=custom:(uuid,obs)`);
```
Comment (rhenshaw56) [inline app/js/components/LabsStatusDisplay.jsx]: can you make this call using an action or in react-component ?

File: app/js/utils/helpers.js [hunk 3]
```diff
@@ -55,6 +55,12 @@ export const filterThrough = (filters, data) => {
     const filteredData = matchSorter(originalData, inputValue, { keys: ['concept.display'] });
     originalData = filteredData;
   }
+
+  if (filters.testStatusField !== "All") {
```
Comment (rhenshaw56) [inline app/js/utils/helpers.js]: I'm not sure if we would need to add filters since the urgency fields themselves don't got filters either

**Comment by jomadoye** [comment]
@rhenshaw56 PR is still in progress.

File: app/js/utils/helpers.js (hunk 3 above)
Comment (jomadoye) [reply app/js/utils/helpers.js]: Dave requested for a filter on the ticket.

**Pull request merged**